content/vendor/guides/airgapped-existing-cluster.md

```diff
@@ -0,0 +1,523 @@
+
+Let's start by downloading the `kots` plugin and the airgap bundle to our workstation:
+
+```shell script
+gcloud compute ssh --ssh-flag=-A ${AIRGAP_JUMP} -- 'curl https://kots.io/install | bash'
+```
+
+Next, let's copy the binary to our airgapped workstation from there
+
+```shell script
+gcloud compute ssh --ssh-flag=-A ${AIRGAP_JUMP} -- \
+  scp /usr/local/bin/kubectl-kots ${AIRGAP_WORKSTATION}:
+gcloud compute ssh --ssh-flag=-A ${AIRGAP_JUMP} -- \
+  ssh ${AIRGAP_WORKSTATION} -- sudo cp kubectl-kots /usr/local/bin
+```
+
+Next we can verify its working
+
+```shell script
+gcloud compute ssh --ssh-flag=-A ${AIRGAP_JUMP} -- \
+  ssh ${AIRGAP_WORKSTATION} -- /snap/bin/kubectl kots version
+```
+
+You should see a version like this:
+
+```text
+Replicated Kots 1.17.2
+```
+
+Next, we need to get the airgap bundle to our workstation. We'll go to https://github.com/replicatedhq/kots/releases and grab the version that matches the installed KOTS cli (1.17.2) in this case, then copy it over
+
+```shell script
+gcloud compute ssh --ssh-flag=-A ${AIRGAP_JUMP} -- \
+  wget https://github.com/replicatedhq/kots/releases/download/v1.17.2/kotsadm.tar.gz
+gcloud compute ssh --ssh-flag=-A ${AIRGAP_JUMP} -- \
+  scp ./kotsadm.tar.gz ${AIRGAP_WORKSTATION}:
+```
+
+Per the docs, we can now push the images using this bundle
+
+```shell script
+gcloud compute ssh --ssh-flag=-A ${AIRGAP_JUMP} -- \
+  ssh ${AIRGAP_WORKSTATION} -- \
+  /snap/bin/kubectl kots admin-console push-images \
+  ./kotsadm.tar.gz ${CLUSTER_PRIVATE_IP}:32000/kotsadm \
+  --registry-username kots \
+  --registry-password kots
+```
+
+Once the images are pushed we can then run the installation. You'll need to grab the Application Slug for your KOTS app, in this case we'll use `sentry-pro`. We'll set a matching namespace as well.
+
+```shell script
+export APP_SLUG=sentry-pro
+export NAMESPACE=sentry-pro
+gcloud compute ssh --ssh-flag=-A ${AIRGAP_JUMP} -- \
+  ssh ${AIRGAP_WORKSTATION} -- \
+  KUBECONFIG=./admin.conf /snap/bin/kubectl kots install \
+  --kotsadm-registry ${CLUSTER_PRIVATE_IP}:32000 \
+  --registry-username kots --registry-password kots \
+  ${APP_SLUG} --namespace ${NAMESPACE} --port-forward=false
+```
+
+**NOTE** if the UI for kots prompts gives you trouble, you may also want to pass `--shared-password=...` to set a password ahead of time.
+
+You should expect to see something like
+
+```text
+
+    Unable to pull application metadata. This can be ignored, but custom branding will not be available in the Admin Console until a license is installed.
+
+  • Deploying Admin Console
+    •  ✓ Creating namespace
+    •  ✓ Waiting for datastore to be ready
+  •  ✓  Waiting for Admin Console to be ready
+
+• To access the Admin Console, run kubectl kots admin-console --namespace sentry-pro
+
+```
+
+### Connecting to KOTS
+
+Now, we'll create a node port to expose the service
```

Comment:
Curious, if this was exposed as a `kubectl port-forward service/kotsadm 8800:3300 --address 0.0.0.0` and then SSH tunnel is ran to that do you still need to configure the `nodeport`?